moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -1348,12 +1346,6 @@ bool moveit::core::RobotState::setFromIK(const JointModelGroup* jmg, const Eigen
   return setFromIK(jmg, pose_in, tip_in, consistency_limits, attempts, timeout, constraint, options);
 }
 
-namespace moveit
-{
-namespace core
-{
-namespace
```

Comment:
please restore the anonymous namespace here (but not the moveit or core namespaces)